src/scales/quantitativeScale.ts

```diff
@@ -152,7 +153,28 @@ export class QuantitativeScale<D> extends Scale<D, number> {
     });
     var newMin = minExistsInExceptions ? min : this.invert(this.scale(min) - (this.scale(max) - this.scale(min)) * p);
     var newMax = maxExistsInExceptions ? max : this.invert(this.scale(max) + (this.scale(max) - this.scale(min)) * p);
-    return this._niceDomain([newMin, newMax]);
+    if (this._niceDomainEnabled) {
+      return this._niceDomain([newMin, newMax]);
+    }
+    return ([newMin, newMax]);
+  }
+
+  /**
+   * Gets whether or not the scale will have a nice domain.
+   */
+  public niceDomain(): boolean;
```

Comment:
This name might be confusing, since it could suggest that it returns something of type `D[]`, similar to `domain()`. Is there a different name we could use?